### PR TITLE
chore(workflows + vulns)

### PR DESCRIPTION
**Pull request by Jose-Matsuda**
See https://github.com/StatCan/aaw-private/issues/121 for information.

The one vulnerability is a [false positive](https://github.com/aquasecurity/trivy/issues/4102)

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -51,6 +52,8 @@ jobs:
     # Scan image for vulnerabilities
     - name: Aqua Security Trivy image scan
       run: |
+        # Ignore Vulnerabilities
+        printf ${{ env.IGNORE_VULNS }} > .trivyignore
```
Comment (Souheil-Yazji) [inline .github/workflows/publish.yml]: Use repo vars for whitelist?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+name: PR_build
```
Comment (Souheil-Yazji) [inline .github/workflows/build.yml:1]: Is this temp or perm?

Comment (Jose-Matsuda) [reply .github/workflows/build.yml:1]: permanent, its so we don't need to push to main everytime we want the workflow to build

File: .github/workflows/publish.yml (hunk 1 above)
Comment (Jose-Matsuda) [reply .github/workflows/publish.yml]: yeah guess i need perms on this

Comment (Jose-Matsuda) [reply .github/workflows/publish.yml]: i suppose the other thing is when we need to update these repo vars for the allowlist, it doesn't keep a history of what's in there so maybe a good feature anyways just to re-evaluate when we need to add or remove them

Comment (Jose-Matsuda) [reply .github/workflows/publish.yml]: donneeeeee

**Review by Souheil-Yazji — APPROVED** [review]
LGTM

**Pull request merged**